File: competitions/onlinejudge/intro/11044/p11044.py
import sys


def min_sonar_count(rows, cols):
    return (rows // 3) * (cols // 3)


if __name__ == '__main__':
    cases = int(sys.stdin.readline())
    for _ in range(cases):
        rows, cols = [int(i) for i in sys.stdin.readline().split()]
        print(min_sonar_count(rows, cols))

File: competitions/onlinejudge/intro/11044/p11044.cpp
#include <iostream>


using namespace std;


int min_sonar_count(int n, int m) {
    return (n / 3) * (m / 3);
}


int main() {

    int t;
    int n, m;
    cin>>t;
    
    for(int i = 0; i<t; i++) {
        cin>>n>>m;
        cout<<min_sonar_count(n, m)<<endl;
    }

    return 0;
}
